docs/_content/ontwikkelaars/algemeen/archiveren.md

```diff
@@ -0,0 +1,70 @@
+# Archiveren (voor dummies)
+
+## Gerelateerde attributen
+
+* `Zaak.Archiefnominatie` Aanduiding of het zaakdossier blijvend bewaard of na een bepaalde termijn vernietigd moet worden:
+
+    * `null`
+    * `Blijvend bewaren`
+    * `Vernietigen`
+    
+* `Zaak.Archiefstatus` De fase waarin het zaakdossier zich qua archivering bevindt:
+
+    * `nog te archiveren` (standaardwaarde)
+    * `gearchiveerd`
+    * `gearchiveerd (procestermijn onbekend)`
+    * `vernietigd`
+    * `overgedragen`
+    
+  Als de `Archiefstatus` `gearchiveerd` of `gearchiveerd (procestermijn onbekend)` is, dan is het zaakdossier niet-wijzigbaar.
+
+* `Zaak.Archiefactiedatum` De datum waarop het gearchiveerde zaakdossier vernietigd moet worden dan wel overgebracht moet worden naar een archiefbewaarplaats. 
+
+* `ResultaatType.Archiefnominatie` Aanduiding die aangeeft of `Zaak`en met een resultaat van dit `ResultaatType` blijvend moeten worden bewaard of (op termijn) moeten worden vernietigd . 
+
+* `ResultaatType.Procestermijn` De periode dat het zaakdossier na afronding van de zaak actief gebruikt en/of geraadpleegd wordt ter ondersteuning van de taakuitoefening van de organisatie.
+
+* `ResultaatType.Archiefactietermijn` De termijn, na het vervallen van het bedrijfsvoeringsbelang, waarna het zaakdossier (de `Zaak` met alle bijbehorende `Informatieobject`en) van een `Zaak` met een resultaat van dit `ResultaatType` vernietigd of overgebracht (naar een archiefbewaarplaats) moet worden. 
+
+
+TODO:
+
+* `Zaak.StartdatumBewaartermijn` (nieuw!) De datum die de start markeert van de termijn waarop het zaakdossier vernietigd moet worden. 
+* `Zaak.Procesobject`
+* `Zaak.Selectielijstklasse`
+* `ResultaatType.BrondatumArchiefprocedure`
+* `ResultaatType.Procesobjectaard`
+* `ZaakType.SelectielijstProcestype`
+* `ZaakType.Archiefclassificatie`
+* `Zaak-InformatieobjectType.Archiefregime`
+* `Zaak-InformatieobjectType.Vernietigingstermijn` (relatie) De termijn waarna informatieobjecten, van het `InformatieobjectType` bij zaken van het `ZaakType` met een resultaat van het `ResultaatType`, vernietigd moeten worden. 
+
+## Berekenen van de `Zaak.Archiefactiedatum`
+
+TODO: Eerste idee maar dit is te simplistisch, want de brondatum is 
+afhankelijk van het `ResultaatType.afleidingswijzeBrondatumArchiefprocedure`.
+Voor de leesbaarheid zetten we dit nu even op de `Zaak.Einddatum`:
+
+`brondatum` = `Zaak.Einddatum`
+
+`Zaak.Archiefactiedatum` = `brondatum` + `Zaak.ResultaatType.Procestermijn` + `Zaak.ResultaatType.Archiefactietermijn`
+
```

Comment:
De bepaling van de procestermijn is het lastigste stuk. Soms is die vooraf bekend, maar kan ook afhankelijk zijn van de levensduur van het procesobject. Hier is een beslisboom voor nodig.

Reply:
Is de procestermijn aan ons (techniek/API) om te bepalen, of iets wat ingevuld ergens moet worden?

Reply:
Procestermijn wordt vastgelegd, in de ZTC lijkt me, zie 2.13.4 `Attribuutsoort Procestermijn`.

@Remkodehaas: ik neem aan dat RGBZ2.0 ook gemodelleerd is? Misschien helpt het Joeri en Sergei als we op basis daarvan een diagrammetje genereren die de relaties toont rondom een aantal ingewikkelde attributen hier? Het gaat dan met name om `Attribuutsoort Archiefactietermijn` (2.18.19) en in mindere mate `Attribuutsoort Procestermijn`. In de documentatie raak ik door alle afhankelijkheden het overzicht een beetje kwijt.

Reply:
> De bepaling van de procestermijn is het lastigste stuk.

Volgens mij valt het wel mee, moet alleen even meer lezen en typen in dit document :)

> In de papieren documentatie raak ik door alle afhankelijkheden het overzicht een beetje kwijt.

Ja :(